Dockerfile

```
@@ -1,44 +1,29 @@
-# Use a specific Python 3.11 slim image as the base
 FROM python:3.11.3-slim
 
-# Set environment variables
 ENV PYTHONDONTWRITEBYTECODE=1 \
     PYTHONUNBUFFERED=1
 
-# Install system dependencies
+# Install minimal system dependencies required for Chrome
 RUN apt-get update && apt-get install -y --no-install-recommends \
```

```suggestion
RUN apt-get update -qq && apt-get install -y --no-install-recommends \
```

quieter docker build